ple/games/donkeykong/Game.py

```diff
@@ -0,0 +1,210 @@
+__author__ = 'Batchu Vishal'
+import pygame
+import sys
+from pygame.locals import *
+from Board import Board
+from .. import base
+import numpy as np
+
+'''
+This class defines the logic of the game and how player input is taken etc
+We run one instance of this class at the start of the game, and this instance manages the game for us.
+'''
+
+
+class DonkeyKong(base.Game):
+    def __init__(self):
+        '''
+        Set the height and width for the game element
+        FPS is set to 30 frames per second
+        '''
+        self.height = 520
+        self.width = 1200
+
+        actions = {
+                "left": K_a,
+                "right": K_d,
+                "jump": K_SPACE,
+                "up": K_w,
+                "down": K_s
+        }
+        base.Game.__init__(self, self.width, self.height, actions=actions)
+        self.rewards = {
+            "positive": 5,
+            "win": 50,
+            "negative": -25,
+            "tick": 0
+        }
+
+    def init(self):
+        # Font is set to comic sans MS
+        self.myFont = pygame.font.SysFont("comicsansms", 30)
```

Comment:
performance: you can keep this in the `__init__` as you shouldnt need to reload it each time the game is reset